local_ads/event.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <chrono>
+#include <string>
+
+namespace local_ads
+{
+using Timestamp = std::chrono::steady_clock::time_point;
+
+enum class EventType
+{
+  ShowPoint = 0,
+  OpenInfo,
+  ClickedPhone,
+  ClickedWebsite
+};
+
+struct Event
+{
+  EventType m_type;
+  int64_t m_mwmVersion;
+  std::string m_countryId;
+  uint32_t m_featureId;
+  uint8_t m_zoomLevel;
+  Timestamp m_timestamp;
+  double m_latitude;
+  double m_longitude;
+  uint16_t m_accuracyInMeters;
+
+  Event(EventType type, int64_t mwmVersion, std::string const & countryId, uint32_t featureId,
+        uint8_t zoomLevel, Timestamp const & timestamp, double latitude, double longitude,
+        uint16_t accuracyInMeters);
+
+  bool operator<(Event const & event) const;
+  bool operator==(Event const & event) const;
+};
+}  // namespace local_ads
+
+extern std::string DebugPrint(local_ads::Event const & event);
```

Comment:
No need in extern keyword. Also, move DebugPrint for Event to the local_ads namespace, as it's a good practice for ADL resolution.